Modules/ITS/src/ITSFeeCheck.cxx

```diff
@@ -137,119 +136,128 @@ std::string ITSFeeCheck::getAcceptedType() { return "TH2I, TH2Poly"; }
 
 void ITSFeeCheck::beautify(std::shared_ptr<MonitorObject> mo, Quality checkResult)
 {
-  TText* tInfo = new TText();
-  TText* tInfoIB = new TText();
-  TText* tInfoML = new TText();
-  TText* tInfoOL = new TText();
-  TText* tInfoLayers[7];
-  TText* tInfoSummary[4];
+  TLatex* tInfo;
+  TLatex* tInfoLayers[7];
+  TLatex* tInfoIB;
+  TLatex* tInfoML;
+  TLatex* tInfoOL;
+  TLatex* tInfoSummary[4];
+  TString status;
+  int textColor;
 
   for (int iflag = 0; iflag < NFlags; iflag++) {
     if (mo->getName() == Form("LaneStatus/laneStatusFlag%s", mLaneStatusFlag[iflag].c_str())) {
       auto* h = dynamic_cast<TH2I*>(mo->getObject());
       if (checkResult == Quality::Good) {
-        tInfo->SetText(0.1, 0.8, "Quality::GOOD");
-        tInfo->SetTextColor(kGreen);
+        status = "Quality::GOOD";
+        textColor = kGreen;
       } else if (checkResult == Quality::Bad) {
-        tInfo->SetText(0.15, 0.8, "Quality::BAD (call expert)");
-        tInfo->SetTextColor(kRed);
+        status = "Quality::BAD (call expert)";
+        textColor = kRed;
       }
-      tInfo->SetTextSize(18);
+      tInfo = new TLatex(0.12, 0.835, Form("#bf{%s}", status.Data()));
```

Comment:
General comment for all `TLatex`: you use `new` so you must delete them when not using them anymore. Otherwise you can avoid `new` here I guess (and hence the `delete`)